tsl/test/isolation/expected/deadlock_drop_chunks_compress.out

```diff
@@ -0,0 +1,237 @@
+Parsed test spec with 2 sessions
+
+starting permutation: s1_drop s1_commit s2_compress_chunk_1 s2_compress_chunk_2 s2_commit
```

Comment:
This iterates over all possible step order combinations, right?

Reply:
Unless a test specifies explicit permutations, it will run all permutations.